.github/workflows/test.yml

```diff
@@ -126,3 +126,31 @@ jobs:
         env:
           TAG: test
           MODULE: Tokens
+  cns-transport-test:
+    name: Run CNS-Transport Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: backbone  
+      - uses: actions/checkout@v3
+        with:
+          repository: nmshd/cns-transport
+          path: cns-transport
+      - run: mkdir shared
+      - run: cp ./backbone/setup-sqlserver.sql shared/setup-sqlserver.sql
```

Comment:
I would suggest to move the two setup scripts into a separate folder "setup-db", rename them to "sqlserver.sql" and "postgres.sql" and then just mount the whole folder into the container. Then you don't need these two steps